Annotations/Crafttweaker_Annotations/src/main/java/com/blamejared/crafttweaker_annotations/annotations/TaggableElement.java

```diff
@@ -0,0 +1,30 @@
+package com.blamejared.crafttweaker_annotations.annotations;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Target(ElementType.TYPE)
+@Retention(RetentionPolicy.RUNTIME)
+public @interface TaggableElement {
+    
+    /**
+     * The ResourceKey location of the registry of this element
+     *
+     * Examples:
+     * - "minecraft:item" for Item
+     * - "minecraft:worldgen/biome" for Biome
+     *
+     * @return The ResourceKey location of the registry of this element.
+     */
```

Comment:
I feel like this Javadoc comment might be better worded. Why not something like "the name of the registry of this element"?